src/leappto/cli.py

```diff
@@ -507,12 +509,19 @@ def _ssh_sudo_out(self, cmd, **kwargs):
             sudo_cmd = "sudo bash -c '{}'".format(cmd)
             return self._ssh_out(sudo_cmd, **kwargs)
 
+        def _get_container_name(self):
+            if self.container_name:
+                return self.container_name
+
+            name_list = ["container", self.source.hostname]
+            return '_'.join(name_list)
+
         def _get_container_dir(self):
-            # TODO: Derive container name from source host name
-            container_name = "container"
+            container_name = self._get_container_name()
             return os.path.join(MACROCONTAINER_STORAGE_DIR, container_name)
 
         def copy(self):
+            container_name = self._get_container_name()
```

Comment:
Please remove lines 531-532. Since we have different names for containers, removing container dir is no needed.